include/kernels/clover_deriv.cuh

```diff
@@ -210,7 +214,7 @@ namespace quda
 
       for (int nu = 0; nu < 4; nu++) {
         if (nu == mu) continue;
-        computeForce<Link>(force, arg, x_cb, parity, mu, nu);
+        computeForce<Link>(force, *this, x_cb, parity, mu, nu);
```

Comment:
Somewhat useless comment, but for whatever reason I don't "like" how `*this` is being passed in as an argument---purely textually. My only suggestion would be to have a `const auto &ftor = *this;` somewhere higher up. I can also just deal, but maybe this bugs other people too?

Reply:
I don't have a strong preference either way.  Maybe we can wait and see what others say.

Reply:
I'm fine with leaving it as-is for this PR, it's far from the first time I've seen this type of thing done via passing `*this`